ApiSecurityInAction/dotnet/NatterApi/Startup.cs

```diff
@@ -31,6 +33,20 @@ public void ConfigureServices(IServiceCollection services)
             services.AddControllers();
 
             services.AddSwaggerGen(c => c.SwaggerDoc("v1", new OpenApiInfo { Title = "NatterApi", Version = "v1" }));
+
+            services.AddAuthentication(CookieAuthenticationDefaults.AuthenticationScheme)
+                .AddCookie(options =>
+                {
+                    options.Cookie.Name = "NatterLoginCookie";
+                    options.LoginPath = "/User/Login";
+                    options.SlidingExpiration = true;
+                    options.Cookie.HttpOnly = true;
+                });
+
+            services.AddAntiforgery(options =>
+            {
+                options.HeaderName = "X-XSRF-TOKEN"; //for those cookies that come from javascript

```

Comment:
Was going to say this is a typo, but I guess apparently `XSRF` and `CSRF` are equivalent, which is news to me lol